.github/pull_request_template.md

```diff
@@ -1,10 +1,21 @@
-Hi! Thanks for contributing to JupyterLab.
-Please look through the checklist below before submitting your pull request.
-
-- [ ] Look through open pull requests to see if someone has already started working on this.
-- [ ] Note what issue(s) this pull request addresses. (We strongly encourage the community to open an issue describing proposed work before opening a pull request.)
-- [ ] Provide a detailed description of the pull request. (Why are these changes necessary? How have they been implemented?)
-- [ ] For UI/UX changes, include before/after screenshots
-- [ ] Include a narrative description of visual or user interaction changes. How does your design effectively address the problem?
-- [ ] Provide a list of Jupyterlab packages the pull request modifies.
-- [ ] Describe any backwards incompatible changes to Jupyterlab’s public APIs.
+<!-- Thanks for contributing to JupyterLab! Please fill out the following items to submit a pull request. -->
+
+## References
+
+<!-- Note here issue numbers this pull request addresses. -->
+
```

Comment:
The "here" makes these read strangly, I'd say remove it from all.  Otherwise looks good

Reply:
I was trying to convey that you should fill in the blank right here. It seems that we've had a lot of issues where people just create from a template, then leave the template at the bottom and start filling in text at the top. The "here" was supposed to tell people to intersperse their text with the template text. How about we take it out for now, and if we keep seeing that issue, we address it?

Reply:
Done!